src/gsIO/gsXmlUtils.hpp

```diff
@@ -415,7 +415,7 @@ class gsXml< gsNurbsBasis<T> >
         
         gsXmlNode * tmp = node->first_node("KnotVector");
         // if type: == Plain, == Compact .. 
-        gsKnotVector<T> kv = * safe( gsXml<gsKnotVector<T> >::get (tmp) );
+        gsKnotVector<T> kv = ( gsXml<gsKnotVector<T> >::get (tmp) );
```

Comment:
This doesn't work. You cannot cast a raw pointer to an object. Moreover, it's a memory leak.

Reply:
Just remove the comment, it is also to old..